.cirrus.yml

```diff
@@ -333,6 +333,45 @@ task:
       on_failure:
         <<: *on_failure_ac
 
+    - name: Linux - Debian Bulsseye - Meson - Test Extensions
+      allow_failures: true
+      depends_on:
+        - Linux - Debian Bullseye - Meson
+
+      configure_script: |
+        su postgres <<-EOF
+          meson setup \
+            --buildtype=debug \
+            --prefix=/usr/local/ \
+            -Dcassert=true \
+            ${LINUX_MESON_FEATURES} \
+            -DPG_TEST_EXTRA="$PG_TEST_EXTRA" \
+            build
+        EOF
+      build_script: su postgres -c 'ninja -C build -j${BUILD_JOBS}'
+      install_script: ninja -C build install
+      upload_caches: ccache
+
+      install_required_packages_for_extensions_script: |
+        apt-get update
+        DEBIAN_FRONTEND=noninteractive apt-get -y install \
+          autoconf \
+          libtool \
+          libgeos-dev \
+          libproj-dev \
+          libprotobuf-c-dev \
+          protobuf-c-compiler
+
+      # some of the make commands of test_extensions script needs to run with sudo,
+      # that command makes postgres user can execute sudo commands without password
+      dont_ask_sudo_pw_script: |
+        echo "postgres ALL=(ALL:ALL) NOPASSWD: ALL" | tee /etc/sudoers.d/postgres
+
+      test_extensions_script: su postgres -c 'python3 src/tools/ci/test_extensions'
```

Comment:
How much work would it be to split this up into one _script per extension, so we can see what works and what fails without scrolling through quite as much text?

Reply:
This is already implemented, .cirrus.yml is updated regarding this comment. Also, I used `always:` to not skip if former test _scripts fail. Is this a best way to do that?